controller/src/beerocks/bml/internal/bml_internal.cpp

```diff
@@ -1266,6 +1266,141 @@ int bml_internal::get_dcs_continuous_scan_enable(const std::string &mac, int *ou
     return send_bml_cmdu(*output_enable, request->get_action_op());
 }
 
+int bml_internal::set_dcs_continuous_scan_params(const std::string &mac, int dwell_time,
+                                                 int interval_time, unsigned int *channel_pool,
+                                                 int channel_pool_size)
+{
+    if (dwell_time == BML_CHANNEL_SCAN_INVALID_PARAM &&
+        interval_time == BML_CHANNEL_SCAN_INVALID_PARAM &&
+        channel_pool_size == BML_CHANNEL_SCAN_INVALID_PARAM) {
+        LOG(ERROR) << "Function is called, but no data is being set!";
+        return (-BML_RET_INVALID_DATA);
+    }
```

Comment:
What if only one of the parameters has invalid value?
Shouldn't this statement use OR conditions?

Reply:
No, this handles the case if no parameters is being requested by the caller
If all params are invalid it means that there is no need to continue with the function
yet, even if one is valid  it will still continue but return only that value

Reply:
Any reason this conversation is unresolved?

Reply:
Forgot to resolve the conversation